synfig-studio/src/gui/states/state_smoothmove.cpp

```diff
@@ -274,7 +274,7 @@ StateSmoothMove_Context::~StateSmoothMove_Context()
 
 
 
-DuckDrag_SmoothMove::DuckDrag_SmoothMove():radius(1.0f)
+DuckDrag_SmoothMove::DuckDrag_SmoothMove():radius(synfigapp::Main::get_selected_input_device()->settings().get_value("smooth_move.radius", 1.0))
```

Comment:
`synfigapp::Main::get_selected_input_device()` may return null, and this results in an exception.

I suggest to move init part to constructor, something like this:
```
DuckDrag_SmoothMove::DuckDrag_SmoothMove():radius(1.0f)
{
	if (const auto device = synfigapp::Main::get_selected_input_device()) {
		radius = device->settings().get_value("smooth_move.radius", 1.0);
	}
```

Reply:
> `synfigapp::Main::get_selected_input_device()` may return null.
oh, I didn't notice this. makes sense to move init to constructor then. done :).